packages/skipper/tests/FileSystem/PathNormalizerTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symplify\Skipper\Tests\FileSystem;
+
+use Iterator;
+use Symplify\PackageBuilder\Testing\AbstractKernelTestCase;
+use Symplify\Skipper\FileSystem\PathNormalizer;
+use Symplify\Skipper\HttpKernel\SkipperKernel;
+
+final class PathNormalizerTest extends AbstractKernelTestCase
+{
+    /**
+     * @var PathNormalizer
+     */
+    private $pathNormalizer;
+
+    public function setUp(): void
+    {
+        $this->bootKernel(SkipperKernel::class);
+        $this->pathNormalizer = self::$container->get(PathNormalizer::class);
+    }
+
+    /**
+     * @dataProvider providePaths
+     */
+    public function testPaths(string $path, string $expected): void
+    {
+        $this->assertEquals(
```

Comment:
My IDE suggests me to use `self::` instead of `$this->` but you used `$this->` in the other tests so leave it as that.